Library/Formula/gyp.rb

```diff
@@ -0,0 +1,24 @@
+class Gyp < Formula
+  desc "Generate Your Projects"
+  homepage "https://gyp.gsrc.io/"
+  url "https://chromium.googlesource.com/external/gyp",
+    :using => :git,
+    :revision => "7d29c3de1f5eb74330b81b78ea116b67f684b23b"
```

Comment:
`gyp` to my knowledge doesn't tag releases, and consequently we've turned it down from the core a couple times before. I'm pretty sure this is still the case?